### PR TITLE
Upgrades

### PR DESCRIPTION
**Pull request by jedesah**

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=0.13.13
+sbt.version=0.13.16-M1
```
Comment (drostron) [inline project/build.properties]: What's the motivation for going with the milestone?

Comment (jedesah) [inline project/build.properties]: Paul Phillips convinced me that this was a good practice in general but in this case specifically, it gives us the cross-publishing functionality necessary to eventually bring `quasar` to use `sbt` `1.0.0`.

**Comment by drostron** [comment]
What's the motivation for `version: breaking` over `version: feature`?

**Comment by jedesah** [comment]
I think `feature` is fine. When I initially put this label, I thought I had included the upgrade of `wartremover` as well which is breaking

**Pull request merged**